definitions/nodes.md

```diff
@@ -1,13 +1,20 @@
 ---
 title: Nodes
 status: Feedback Appreciated
-category: concept
+category: Concept
 ---
+
 ## Nodes
 
-A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a VM, or even a container.
+### What it is
+
+A node is a computer that works in concert with other computers, or nodes, to accomplish a common task. When we talk about cloud native platforms a node could be a physical computer, a virtual computer, referred to as a [VM](virtual_machine.md), or even a [container](container.md).
+
+### Problem it Addresses
+
+A node exists to run apps or processes as part of a group of computers. A node is part of a [cluster](cluster.md), or group, of nodes that works together in concert.
 
-A node exists to run apps or processes as part of a group of computers. A node is part of a cluster, or group, of nodes that works together in concert.
+### How it Helps
 
-A node gives you a distinct unit of compute, memory/cpu/network, that you can assign to a cluster. In a cloud native platform or app a node represents a single unit that can perform work. Ideally, individual nodes are undifferentiated in that any one node of a particular type is indistinguishable from any other node of the same type.
+A node gives you a distinct unit of compute, memory/cpu/network, that you can assign to a [cluster](cluster.md). In a cloud native platform or app a node represents a single unit that can perform work. Ideally, individual nodes are undifferentiated in that any one node of a particular type is indistinguishable from any other node of the same type.
```

Comment:
This sounds more like the definition of a virtual machine. I guess for "node" we need something more like "By connecting multiple nodes, developers create a [distributed system](distributed_systems.md) where apps can run more reliably...." @JasonMorgan, let's flesh this out a little.

Reply:
I disagree. I think the text is good.